src/docs/markdown/caddyfile/directives/tracing.md

```diff
@@ -0,0 +1,53 @@
+---
+title: tracing (Caddyfile directive)
+---
+
+# tracing
+
+It provides integration with OpenTelemetry tracing facilities.
+
+When enabled, it will propagate an existing trace context or initialize a new one.
+
+It is based on `https://github.com/open-telemetry/opentelemetry-go`.
+
+It uses `gRPC` as an exporter protocol and  W3C `tracecontext` and `baggage` as propagators.
```

Comment:
Possibly link to relevant docs (I don't know what these are, lol)

Reply:
I agree 😃  Links added 👍